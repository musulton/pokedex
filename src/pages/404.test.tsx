import {render} from "@testing-library/react";

import NotFound from "@/pages/404";

it("should render NotFound correctly", () => {
    const {container} = render(<NotFound />)

    expect(container).toMatchSnapshot();
})
