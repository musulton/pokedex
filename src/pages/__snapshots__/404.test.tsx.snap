// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound correctly 1`] = `
<div
  class="sc-249c5c88-0 bHOtmB"
>
  <p
    class="sc-249c5c88-1 dUHQTv"
  >
    404 | Resource not found...
  </p>
</div>
`;
